.travis.yml

```diff
@@ -24,11 +24,10 @@ matrix:
     - php: 5.6
       env: SYMFONY_VERSION=2.3.* SYMFONY_DEPRECATIONS_HELPER=weak
     - php: 5.6
-      env: SYMFONY_VERSION=2.8.*
+      env: SYMFONY_VERSION=2.8.* SYMFONY_DEPRECATIONS_HELPER=99999
```

Comment:
Not really sure what you do here, as this is closer to "weak" than "strict". I would just set this to `strict` to see all deprecation notices